lib/rpcc_dpdk/rte_flow_ndn.c

```diff
@@ -842,3 +842,164 @@ rpc_rte_flow_release_united_items(rcf_rpc_server       *rpcs,
 
     RETVAL_VOID(rte_flow_release_united_items);
 }
+
+static const char *
+tarpc_rte_flow_indir_action_conf2str(
+    const struct tarpc_rte_flow_indir_action_conf *conf,
+    te_string *conf_str)
+{
+    if (conf_str == NULL)
+        return "{ NULL }";
```

Comment:
log_buff must be used anyway, typically NULL is represented as {} or just NULL.. { NULL } does not make sense.